pom.xml

```diff
@@ -135,6 +136,7 @@
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>cobertura-maven-plugin</artifactId>
         <version>${cobertura.maven.plugin.version}</version>
+        <inherited>false</inherited>
```

Comment:
I guess this just means this plugin isn't available for use in child projects right? So then it would only apply to this parent module where it's irrelevant since this is a pom with no Java code or tests to report coverage on. So I think we should either just remove it entirely (which I'm fine with) or we look into using <reportSets> to control how its used downstream.

Reply:
you're right, so as it's fine with you, I removed it and added this change to the changelog.